cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -471,6 +471,41 @@ action.cyences_notable_event_action.products = Sophos Endpoint Protection
 action.cyences_notable_event_action.teams = SOC
 
 
+[Sophos Endpoint Protection - Firewall Lost Connection to Sophos Central]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 54 * * * *
+description = This alert will trigger when a Firewall lost connection to Sophos Central. \
+\
+Data Collection - Sophos Central Add-on for Splunk (https://splunkbase.splunk.com/app/6186/)
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_sophos` sourcetype="sophos_events" type="Event::Firewall::LostConnectionToSophosCentral" \
+| stats count, latest(_time) as _time, values(name) as threat, values(source_info.ip) as src_ip by host, location | sort -count \
+| eval cyences_severity = "critical" \
```

Comment:
Have we verified the severity with the requester??

Reply:
verified and updated it to high.